src/app/zap-templates/encoder-src.zapt

```diff
@@ -197,71 +57,67 @@ uint16_t encodeApsFrame(uint8_t * buffer, uint16_t buf_length, EmberApsFrame * a
                             apsFrame->options, apsFrame->groupId, apsFrame->sequence, apsFrame->radius, !buffer);
 }
 
-uint16_t _encodeCommand(BufBound & buf, EndpointId destination_endpoint, ClusterId cluster_id, uint16_t mfgCode, CommandId command,
-                        uint8_t frame_control)
-{
-    CHECK_FRAME_LENGTH(buf.Size(), "Buffer is empty");
+#define COMMAND_HEADER(name, clusterId)                                                                                            \
+    const char * kName = name;                                                                                                     \
+                                                                                                                                   \
+    PacketBufferHandle payload = PacketBuffer::NewWithAvailableSize(kMaxBufferSize);                                               \
+    if (payload.IsNull())                                                                                                          \
+    {                                                                                                                              \
+        ChipLogError(Zcl, "Error encoding %s command", name);                                                                      \
+        return PacketBufferHandle();                                                                                               \
```

Comment:
Could just `return payload` too.  Might end up smaller codesize, since it can just jump to the end-of-function return.